spring-addons-starter-rest/src/test/java/com/c4_soft/springaddons/rest/reactive/SpringAddonsServerWebClientSupportTest.java

```diff
@@ -0,0 +1,81 @@
+package com.c4_soft.springaddons.rest.reactive;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.matching.ContainsPattern;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.oauth2.jwt.Jwt;
+import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationToken;
+import org.springframework.security.test.context.TestSecurityContextHolder;
+import org.springframework.security.test.context.support.ReactorContextTestExecutionListener;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestExecutionListener;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.wiremock.spring.EnableWireMock;
+import org.wiremock.spring.InjectWireMock;
+import reactor.test.StepVerifier;
+
+
+@ActiveProfiles("forward-bearer")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@EnableWireMock
+class SpringAddonsServerWebClientSupportTest {
```

Comment:
This `SpringAddonsServerWebClientSupportTest` and `SpringAddonsServerWebClientSupportAnonymousTest` should probably be merged into a single test class with different test cases.